cpu/stm32l1/periph/i2c.c

```diff
@@ -85,27 +81,31 @@ int i2c_init_master(i2c_t dev, i2c_speed_t speed)
             return -2;
     }
 
-    /* read static device configuration */
     switch (dev) {
-#if I2C_0_EN
+
         case I2C_0:
-            i2c = I2C_0_DEV;
-            port = I2C_0_PORT;
-            pin_scl = I2C_0_SCL_PIN;
-            pin_sda = I2C_0_SDA_PIN;
-            I2C_0_CLKEN();
             I2C_0_PORT_CLKEN();
-            NVIC_SetPriority(I2C_0_ERR_IRQ, I2C_IRQ_PRIO);
-            NVIC_EnableIRQ(I2C_0_ERR_IRQ);
             break;
-#endif
+
+        case I2C_1:
+            I2C_1_PORT_CLKEN();
+            break;
```

Comment:
@PeterKietzmann Yes it's done [here](https://github.com/RIOT-OS/RIOT/blob/master/cpu/stm32l1/periph/gpio.c#L80).